thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/views/AnalysisView.js

```diff
@@ -39,18 +41,27 @@ AnalysisView.prototype = {
       }
     });
 
-    analysisMetricSelect.on("change", function(e) {
-      var selectedElement = $(e.currentTarget);
-      var selectedData = selectedElement.select2("data");
-      var metricId = selectedData.map(function (e) {return e.id})[0];
-      var metricAlias = selectedData.map(function (e) {return e.text})[0];
-      var metricName = selectedData.map(function (e) {return e.name})[0];
-      self.viewParams['metric'] = {id: metricId, alias: metricAlias, allowClear:true, name:metricName};
+    analysisMetricSelect.on('change', (e) => {
```

Comment:
Airbnb's guideline is to omit parenthesis when braces are also omitted, otherwise keep them for clarity https://github.com/airbnb/javascript#arrows--one-arg-parens
Are we doing this differently ?